Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -252,6 +253,65 @@ def ip_to_indicator_type(ip):
             return None
 
 
+def auto_detect_indicator_type(indicator_value):
+    """
+      Infer the type of the indicator.
+
+      :type indicator_value: ``str``
+      :param indicator_value: The indicator whose type we want to check. (required)
+
+      :return: The type of the indicator.
+      :rtype: ``str``
+    """
+    try:
+        import tldextract
+    except Exception:
+        raise Exception("Missing tldextract module, In order to use the auto detect function please use a docker"
+                        " image with it installed such as: demisto/jmespath:1.0.0.6980")
```

Comment:
the tag (1.0.0.6980 is redundant,
```suggestion
                        " image with it installed such as the latest tag of: demisto/jmespath")
```